cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -410,20 +407,19 @@
           of the steps is not significant in cases where there are no
           dependencies between the inputs and outputs of the steps.  If any of
           the listed steps fail, then the CWT MUST be rejected -- that is,
-          treated by the application as an invalid input.
+          treated by the application as invalid input.
         </t>
         <t>
           <list style="numbers">
             <t>
               Verify that the CWT is a valid CBOR object.
             </t>
             <t>
-              If the object begins with the CWT CBOR tag, remove it
-	      and verify that one of the COSE CBOR tags follows it.
-	    </t>
-	    <t>
-	      If the object is tagged with one of the COSE CBOR tags, remove it
-	      and verify that it corresponds to the structure of the following COSE object.
+              If the object begins with the CWT CBOR tag, remove it.
+	          </t>
+	          <t>
+	            Read the COSE CBOR tag, remove it and verify that the CBOR object
+              corresponds the indicated COSE CBOR tag.
```

Comment:
I disagree with this change for the same reason as in the previous comment.  We should keep things parallel.

Also, this wording assumes that there is a COSE CBOR tag, which will not always be present.

Please revert this part of the change.